modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Default.js

```diff
@@ -1,8 +1,16 @@
 // If a style has been marked `def`, we need to make sure it
-// and it's attributes are prioritized in our editor.
-// It's easier to create a new Node with our specifications and put it
-// at the front of the line than try to infer between editor instantiation
-// and the editor-user setting styles via the toolbar to know when its needed.
+// and its attributes are prioritized in our editor.
+//
+// Specifically, when the "enter" key is pressed, we don't want
```

Comment:
I did my best to beef up this already good summary, we'll see if it helps us next time 😆

Reply:
much more comprehensible 😅